src/components/SecondaryNavigation/SecondaryNavigationItem.theme.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+import { createStyledTag, createComponentTheme } from '../../utils';
+
+const name = 'secondaryNavigationItem';
+
+const rootTheme = createComponentTheme(name, ({ COLORS }: *) => ({
+  root: {
+    height: '32px',
+    display: 'flex',
+    backgroundColor: 'inherit',
+    textDecoration: 'none',
+    userSelect: 'none',
+    position: 'relative',
```

Comment:
I think not visible styles like `position`, `display`, `userSelect` and others should not be in the theme object. Prefer to describe them in the `createStyledTag`